### PR TITLE
docs: Add global settings doc for admin > config

### PR DESCRIPTION
**Pull request by ryan-blunden**
Preview at https://docs.sourcegraph.com/@rb-docs-global-settings/admin/config/settings

### CODE REVIEW

File: doc/admin/config/global_settings.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Global settings
```
Comment (dadlerj) [inline doc/admin/config/global_settings.md]: Should this just be "Settings" and then describe the different levels/the cascade below?

File: doc/_resources/templates/document.html [hunk 2]
```diff
@@ -134,6 +134,7 @@
                             <li class="content-nav-no-hover">
                                 <ul class="content-nav-section-subsection">
                                     <li><a href="/admin/config/site_config">Site configuration</a></li>
+                                    <li><a href="/admin/config/global_settings">Global settings</a></li>
```
Comment (dadlerj) [inline doc/_resources/templates/document.html]: Same

Comment (dadlerj) [inline doc/_resources/templates/document.html]:
```suggestion
                                    <li><a href="/admin/config/global_settings">Global settings</a> (UI and user-specific customization)</li>
```

File: doc/admin/config/global_settings.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Settings provide the ability to customize and control the Sourcegraph UI and user specific features. They do not configure operational aspects of the instance.
```
Comment (dadlerj) [inline doc/admin/config/global_settings.md]:
```suggestion
Settings provide the ability to customize and control the Sourcegraph UI and user-specific features. They do not configure operational aspects of the instance (which are set in the [site configuration](site_config.md)).
```

File: doc/admin/config/global_settings.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+Settings are also different to [site configuration](site_config.md) as they are set the global level by site admins, the organization level by organization members, and at the individual user level.
```
Comment (dadlerj) [inline doc/admin/config/global_settings.md]:
```suggestion
Settings can be set at the global level (by site admins), the organization level (by organization members), and at the individual user level.
```

**Comment by dadlerj** [comment]
Also — perhaps also add a link at the top of the site-config.md page that links to this one (similar to how this one links to it)?

File: doc/admin/config/global_settings.md (hunk 1 above)
Comment (ryan-blunden) [reply doc/admin/config/global_settings.md]: I wasn't sure about this, but I think changing to just Settings is right.

**Comment by unknwon** [comment]
Thanks for the docs!

> Preview at [docs.sourcegraph.com/@rb-docs-global-settings/admin/config/global_settings](https://docs.sourcegraph.com/@rb-docs-global-settings/admin/config/global_settings)

Hmm, this is 404 🤔

**Comment by ryan-blunden** [comment]
> Hmm, this is 404 🤔

I changed the file name and forgot to update the description. The preview link is https://docs.sourcegraph.com/@rb-docs-global-settings/admin/config/settings

**Pull request merged**